mmyolo/models/task_modules/assigners/batch_dsl_assigner.py

```diff
@@ -146,7 +146,8 @@ def forward(self, pred_bboxes: Tensor, pred_scores: Tensor, priors: Tensor,
             assigned_labels=assigned_labels,
             assigned_labels_weights=assigned_labels_weights,
             assigned_bboxes=assigned_bboxes,
-            assign_metrics=assign_metrics)
+            assign_metrics=assign_metrics,
+            matched_gt_inds=matched_gt_inds)
```

Comment:
Why do we need to change?

Reply:
we need matched_gt_inds to find the labeled bounding box corresponding to the positive samples. Do you know a simpler implementation？

Reply:
I guess I can use assigned_bboxes to find the corresponding labeled bounding box, I'll try it later.